test/parallel/test-gc-http-client.js

```diff
@@ -10,44 +10,40 @@ function serverHandler(req, res) {
 }
 
 const http = require('http');
-const todo = 500;
+const todo = 300;
 let done = 0;
 let count = 0;
 let countGC = 0;
 
 console.log(`We should do ${todo} requests`);
 
 const server = http.createServer(serverHandler);
-server.listen(0, getall);
-
+server.listen(0, () => {
```

Comment:
Can you wrap this in `common.mustCall()`?

Reply:
We have lots of tests that make sure that `server.listen` triggers the callback. If that would not hold, hundreds of tests would fail but if you think strongly about it, I'm going to add it.

Reply:
@BridgeAR I don't feel strongly about it :)